DESCRIPTION

```diff
@@ -10,8 +10,12 @@ Description: Provides a single interface to many sources of full text
 Version: 0.1.9.9211
 License: MIT + file LICENSE
 Authors@R: c(
-    person("Scott", "Chamberlain", role = c("aut", "cre"), email = "myrmecocystus@gmail.com"),
-    person("Will", "Pearse", role = "ctb")
+    person("Scott", "Chamberlain",
+        role = c("aut", "cre"),
+        email = "myrmecocystus@gmail.com",
+        comment = c(ORCID = "orcid.org/0000-0003-1444-9135")),
```

Comment:
@graceli8 Looks like CRAN wants the orcid to be just the number without the url, i.e. `ORCID = "0000-0003-1444-9135"`.  Can you fix that?  Thanks!